literature.bib

```diff
@@ -6861,6 +7065,89 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{XZH+:SOSP13,
+	author = {Xu, Tianyin and Zhang, Jiaqi and Huang, Peng and Zheng, Jing and Sheng, Tianwei and Yuan, Ding and Zhou, Yuanyuan and Pasupathy, Shankar},
+	title = {{Do No Blame Users for Misconfigurations}}, 
+	booktitle = SOSP,
+	year = 2013,
+	pages = {244--259},
+	publisher = ACM,
+	address = NY,
+	url = {https://doi.org/10.1145/2517349.2522727},
+	doi = {10.1145/2517349.2522727},
+	numpages = {16},
+	keywords = {inference, constraint, vulnerability, testing, misconfiguration},
+	location = {Farminton, Pennsylvania},
+}
+
+@inproceedings{NDT+:MSR13,
+	author = {Nadi, Sarah and Dietrich, Christian and Tartler, Reinhard and Holt, Richard C. and Lohmann, Daniel},
+	title = {{Linux Variability Anomalies: What Causes Them and How Do They Get Fixed?}}, 
+	booktitle = MSR,
+	year = 2013,
+	pages = {111--120},
+	publisher = IEEE,
+	url = {https://doi.org/10.1109/MSR.2013.6624017},
+	doi = {10.1109/MSR.2013.6624017},
+	address = Washington
+} 
+
+@inproceedings{SRS:AOSD13,
+	author = {Schulze, Sandro and Richers, Oliver and Schaefer, Ina},
+	title = {{Refactoring Delta-oriented Software Product Lines}}, 
+	booktitle = AOSD,
+	year = 2013,
+	pages = {73--84},
+	publisher = ACM,
+	address = NY,
+	url = {https://doi.org/10.1145/2451436.2451446},
+	doi = {10.1145/2451436.2451446},
+	numpages = {12},
+	keywords = {software product lines, refactoring, evolution, delta-oriented programming},
+	location = {Fukuoka, Japan},
+} 
+
+@book{ABK+13,
```

Comment:
book is definitely already in here

please double check to not add any duplicates

Reply:
Yes the entry already exists. I removed Michael's entry but merged ISBN and URL to the existing entry.